<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1800</width>
    <height>1225</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="connectorTable">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>1351</width>
      <height>801</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>3</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>4</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>5</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>1</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>2</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>3</string>
     </property>
    </column>
   </widget>
   <widget class="QSpinBox" name="RowSpinBox">
    <property name="geometry">
     <rect>
      <x>1690</x>
      <y>100</y>
      <width>73</width>
      <height>49</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="ColumnSpinBox">
    <property name="geometry">
     <rect>
      <x>1690</x>
      <y>20</y>
      <width>73</width>
      <height>49</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="SetConfigButton">
    <property name="geometry">
     <rect>
      <x>1600</x>
      <y>1010</y>
      <width>170</width>
      <height>48</height>
     </rect>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="channelTable">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>840</y>
      <width>1761</width>
      <height>131</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>channels</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>1</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>2</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>3</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>4</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>5</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>6</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>7</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="SetSaveButton">
    <property name="geometry">
     <rect>
      <x>1600</x>
      <y>1080</y>
      <width>170</width>
      <height>48</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1440</x>
      <y>20</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Change Columns:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1440</x>
      <y>100</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Change Rows:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="LengthSpinBox">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>990</y>
      <width>73</width>
      <height>49</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>990</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Change Length:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fileNameEdit">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>1010</y>
      <width>701</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="BrowseButton">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>1080</y>
      <width>241</width>
      <height>48</height>
     </rect>
    </property>
    <property name="text">
     <string>set Map File</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1800</width>
     <height>39</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
